Registry/Registry.EM_COMMON

```diff
@@ -412,8 +412,10 @@ state    real   tratx           ij      misc        1         -      r
 state    real   obs_savwt      hikj     dyn_em      1         X      -       "OBS_SAVWT"           "Internal space holding weights of each ob, for each i,j,k" "dimensionless"
 
 # Other
-state   real    rdx            -        misc      -         -     irh       "rdx"                   "INVERSE X GRID LENGTH"         ""      
-state   real    rdy            -        misc      -         -     irh       "rdy"                   "INVERSE Y GRID LENGTH"         ""      
+state   real    rdx            -        misc      -         -     irh       "rdx"                   "INVERSE X GRID LENGTH"         "m-1"      
+state   real    rdy            -        misc      -         -     irh       "rdy"                   "INVERSE Y GRID LENGTH"         "m-1"      
+state   real    dx2d           ij       misc      -         -     rh        "dx2d"                  "sqrt(dx*mf * dy*mf)"           "m"
+state   real    area2d         ij       misc      -         -     rh        "area2d"                "dx*mf * dy*mf"                 "m2"
```

Comment:
Should this be `dx/mf * dy/mf`? Even with this correction, maybe a description that says what the field represents would be more helpful to users? E.g., `dx2d` could be `2-d GRID DISTANCE` and `area2d` could be `GRID CELL AREA`? If someone really cares exactly how these are computed, they should be able to easily find the computation in the code.

Reply:
Michael,
Yes, the definitions SHOULD have been dx/mf not dx*mf. 
I took you suggestion, and made the description of dx2d and area2d an actual description, instead of an equation. Providing a description is conventional.